.github/workflows/check_api_docs.yml

```diff
@@ -0,0 +1,53 @@
+name: check-api-docs
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: 'Project to check'
+        required: true
+        default: 'csip'
+        type: choice
+        options:
+          - activities
+          - adjudications
+          - alerts
+          - casenotes
+          - court-sentencing
+          - csip
+          - locations
+          - nomis-mapping-service
+          - nomis-prisoner
+          - non-associations
+          - organisations
+          - personal-relationships
+          - sentencing-adjustments
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build:
+    runs-on: [ self-hosted, hmpps-github-actions-runner ]
```

Comment:
this line means it will run in cloud platform, so will have access to the dev servers